src/app/modules/auth/components/register-form/register-form.component.html

```diff
@@ -0,0 +1,52 @@
+<form action="">
+  <mat-form-field>
+    <input matInput placeholder="First Name" required name="firstName" #firstName
+      [(ngModel)]="userRegisterRequest.firstName" [ngModelOptions]="{standalone: true}"
+      (blur)="nameChanged('firstName',$event)">
+    <mat-icon matSuffix>perm_identity</mat-icon>
+    <mat-error *ngIf="userRegisterFromError.firstName">
+      {{userRegisterFromError.firstName}}
```

Comment:
Do we need this model check here? Directly validating from form model object will work right?